docker/addons/journal/docker-compose.yaml

```diff
@@ -16,7 +16,6 @@ networks:
 volumes:
   supermq-journal-volume:
 
-
```

Comment:
Keep this one. One empty line is for the value (empty), and the other isthe  separator for better readability.